src/templates/shared/wrapper.js

```diff
@@ -1,27 +1,28 @@
-const {html, alert, ol, pageAnchor} = require("./bits");
+const {html, alert, escapeHtml, REPO_URL, DISCORD_URL, ul, anchor} = require("./bits");
+const footer = require("./footer");
+const header = require("./header");
+const breadcrumbs = require("./breadcrumbs");
+const toc = require("./toc");
 
-const DISCORD_URL = "https://discord.reclaimers.net";
-const REPO_URL = "https://github.com/Sigmmma/c20";
+const TOC_MIN_HEADERS = 2;
+
+const topLevelTopics = [
+  ["/blam/tags", "Tags"],
+  ["/games/h1", "Halo"]
+];
```

Comment:
This is really just a placeholder for now, I'd like to include more content in the sidebar like child pages and more top-level topics. There's issues for that in zube.